<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <widget class="QWidget" name="m_blurEffect" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>391</width>
     <height>591</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Envoyer à:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rcpt"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>57</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sujet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="subject"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Pièce(s) jointe(s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="file">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseBtn">
          <property name="text">
           <string>Rechercher</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Message:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid grey; border-radius: 5px;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="msg">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>350</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid black; border-radius: 5px;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="DBlurEffectWidget" name="settings" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>590</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>590</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Paramètres</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
    <widget class="Avatar" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/2017-11-02 15.41.47.jpg); background-position: top-right;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lSMTP2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>aucun SMTP</string>
     </property>
    </widget>
    <widget class="QLabel" name="pSMTP2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>995</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>imap.gmail.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>465</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Serveur IMAP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="server">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>smtp.gmail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Serveur SMTP:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Adresse:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="uname">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>example@example.example</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="paswd">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 90); color: white;</string>
     </property>
     <property name="text">
      <string>example</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Mot de passe:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logo1" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png) 0 0 0 0 stretch stretch;</string>
    </property>
    <property name="locale">
     <locale language="French" country="France"/>
    </property>
    <property name="interactive" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="DBlurEffectWidget" name="bar" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>300</width>
     <height>590</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>590</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Mails</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Déconnecté</string>
     </property>
    </widget>
    <widget class="Avatar" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/2017-11-02 15.41.47.jpg); background-position: top-right;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lSMTP1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>aucun SMTP</string>
     </property>
    </widget>
    <widget class="QLabel" name="pSMTP1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="setBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>Paramètres</string>
    </property>
   </widget>
   <widget class="QWidget" name="logo2" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png) 0 0 0 0 stretch stretch;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>Répertoire(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runTest">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string>run Test</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; background-color: rgba(0, 0, 0, 75);</string>
    </property>
   </widget>
  </widget>
  <widget class="DBlurEffectWidget" name="MailShower" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>21</width>
     <height>591</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="MailCont"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>bar</zorder>
  <zorder>settings</zorder>
  <zorder>m_blurEffect</zorder>
  <zorder>MailShower</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DBlurEffectWidget</class>
   <extends>QWidget</extends>
   <header>dblureffectwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Avatar</class>
   <extends>QWidget</extends>
   <header>avatar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
